### ISSUE TITLE
RESIDENT EVIL REVELATIONS

### ISSUE THREAD

**Issue by xx1182** [skyline-emu/skyline-games-list#1106]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

CI #1353

### Android Version

12

### SoC type

Snapdragon

### Device GSMArena link

https://m.gsmarena.com/samsung_galaxy_z_fold3_5g

### Game Behaviour

It doesn't seem to boot at all

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

https://drive.google.com/file/d/1herljzVPVYk9-BukqgiZpKwDDUTpajzC/view?usp=drivesdk

**Comment by TheASVigilante** [comment]
Seems like a vfs issue

**Comment by xx1182** [comment]
To make the issue seems to be some sort of decryption problem:
On PC emulator Yuzu: 
when you start and press AAAAAA at boot chiptune plays and it doesn't boot.
When you start and wait, it says panic press a for a second and then boots.

**Comment by xx1182** [comment]
ETA? I speak from my ass so I'm entitled to ask.
Just bear with my frustration, you don't have to reply, I have countless ways of playing this game, I even got an actual switch;

This seems like a big feature, since usually Capcom Titles set the Bar for future developments:
Think like GTA series on PS2:
-3 Is like at the start 
-vice city is medium(greatly improved light, pyshics, etc)
-san Andreas is pushing to the limits

Maybe fixing this fixes many more. Regards.

**Comment by TheASVigilante** [comment]
Well the filesystem is currently very much incomplete and a lot of games have issues with it however currently gpu accuracy is prioritised over at the moment so it's currently on hold. I could take a look at this for you but it might take a while.

**Comment by xx1182** [comment]
> Well the filesystem is currently very much incomplete and a lot of games have issues with it, however currently gpu accuracy is prioritised over it at the moment so it's currently on hold. I could take a look at this for you but it might take a while.

On a similar note:
I've tried this game on 2 emulators:
1 Yuzu PC, 15-20 fps, playable Reay badly...
2 Competitor Android Emulator, as far as I know the competitor stole some Yuzu code and that's probably the most legal part: Completely playable, 30 or 60fps(i can't remember but Native fps).

PC Ryzen 2200u
Android Latest Snapdrsgon

Am I underestimating smartphones? Is it simply easier to virtualize an instruction set rather than a whole architecture? Idk, but I wish to. Is the competitor using acient Asian magic?

**Comment by TheASVigilante** [comment]
No, merely it's just that emulating the CPU is REEEAAALLYY expensive, for one instruction yuzu (and any other x86 switch emu) has to do at least 6-10 instructions meanwhile your doesn't have to emulate it at all, it runs on the same CPU instructions as your switch does.

**Comment by xx1182** [comment]
### Tested Build

CI #1376

### Game Behaviour
It boots but it presents major graphical issues. I've also tried the sequel Revelations2 which has less graphical issues, but it's still unplayable

### Log File
https://drive.google.com/file/d/1qZDydlFAUnz7Zt_FYxtRwmOwOgOi1lGK/view?usp=drivesdk

**Comment by xx1182** [comment]
Updated.
